### ISSUE TITLE
Remove containers

### ISSUE THREAD

**Issue by johrstrom** [OSC/ood-images#23]
Containers should be built from the repo itself not here. We can keep the vagrant or other VM stuff here, we just just remove container stuff.

**Comment by treydock** [comment]
The containers here aren't for distribution, at least not the ones I've used.  They are for testing , like multi node Slurm cluster using containers.  It's way faster to do that with containers than Vagrant. I only use Vagrant for testing SELinux as can't use SELinux in containers.

**Comment by johrstrom** [comment]
This discourse topic prompted it. The user tried the regular container, then found this repo.  They're not being maintained - even if they're not for distribution someone may stumble upon them.

https://discourse.openondemand.org/t/open-ondemand-is-currently-offline-for-maintenance-after-stoping-and-starting-docker-container/2049

**Comment by treydock** [comment]
Rather than remove things that are useful for developers, we just add some kind of big warning to the README that these are not for use outside OSC or move the containers to a different repo with some kind of name to make it clear, don't use them.  Just don't want to remove something useful because people use Google to find things rather than reading documentation (assuming such documentation exists).

**Comment by treydock** [comment]
Also I'll add that these containers , especially the multi-node ones, allow submitting actual Slurm jobs and testing that capability inside the container.  Getting rid of something that useful seems wrong to me.  If the issue is they are unmaintained, maybe just need maintain them more?

**Comment by johrstrom** [comment]
> Also I'll add that these containers , especially the multi-node ones, allow submitting actual Slurm jobs and testing that capability inside the container

Agreed - but I kinda started directing folks here. I could be convinced that we keep compose files maintained here. 

https://github.com/ubccr/hpc-toolset-tutorial
